nservicebus/upgrades/7to8/index.md

```diff
@@ -11,6 +11,21 @@ upgradeGuideCoreVersions:
 
 NOTE: This is a working document, as there is currently no plan or timeline for the release of NServiceBus version 8.0.
 
+## Support for external dependency injection containers
+
+Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the [`NServiceBus.Extensions.DependencyInjection` package](/nservicebus/dependency-injection/extensions-dependencyinjection.md) provides the ability to use any container that conforms to the `Microsoft.Extensions.DependencyInjection` container abstraction.
+
+The following adapter packages will no longer be provided:
+
+* [Autofac](/nservicebus/dependency-injection/autofac.md)
+* [Castle](/nservicebus/dependency-injection/castlewindsor.md)
+* [StructureMap](/nservicebus/dependency-injection/structuremap.md)
+* [Unity](/nservicebus/dependency-injection/unity.md)
+
+### Property injection
+
+NServiceBus Version 7 enabled [automatic property injection](/nservicebus/dependency-injection/property-injection.md), this is not possible via `Microsoft.Extensions.DependencyInjection` and must be explicitly enabled using the native API of containers that supports it.
```

Comment:
This will be pointing to a different link, right?
That means we'd need to set up a redirect for `property injection` to ensure those that search for it get to the new page.